magento/magento2
# Fix #22956 - make country names translatable

### Description (*)
This is fix for #22956 

### Fixed Issues (if relevant)

1. magento/magento2#22956: Unable to translate checkout country names

### Manual testing scenarios (*)

1. Set up locale mk_MK for store view
2. Add translations file: app/design/frontend/Vendor/theme/i18n/mk_MK.csv with content:
```
"Country","Country Translated"
"Северна Македонија","severna makedonija"
```
3. Flush cache.
4. Add product to cart and go to checkout shipping address form
5. Both country select label **and** single country name should be translated


### Contribution checklist (*)
